### ISSUE TITLE
Voiceover incorrectly expands "Gov." to "Governor"

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#9]
**Upstream bug:** https://openradar.appspot.com/radar?id=4938627352100864
**Date:** 2016-11-24
**Reported by:** Oliver Byford
**Related to:** -

---

## Overview

Voiceover automatically expands "Gov." to "Governor" – presumably to handle cases like "Gov. Jerry Brown". However, it does this even if the period is followed by further alphabetical characters as in "GOV.UK" or "GOV.AU" – which seems to be a common pattern for (e.g.) government websites.

## Steps to Reproduce:
1. Enable Voiceover on iOS 10.1
2. Visit https://www.gov.uk/
3. Instruct Voiceover to read the page content.

## Expected Results:
Voiceover would read:

> [Skip to main content, link]
> Gov UK, [link, banner, landmark]
> Welcome to Gov UK
> The best place to find government services and information
> Simpler, clearer, faster
> Search Gov UK
> [...]

Or a sensible variant thereof (e.g. pronouncing the period in GOV.UK as it would in a url)

## Actual Results:
Voiceover reads:

> [Skip to main content, link]
> Governor, UK, [link, banner, landmark]
> Welcome to Governor, UK
> The best place to find government services and information. Simpler, clearer, faster.
> Search Governor, UK.
> [...]

## Version:
iOS 10.1 [14B72]

## Notes:
Cannot reproduce on on OS X 10.11.6 (15G1108) or macOS 10.12.2b (16C48b) – this only appears to happen on iOS.

We believe (anecdotally) that this may have been happening since at least iOS 9.

## Configuration:
iPhone 6s, Voiceover enabled, Daniel (Enhanced) Voice

**Comment by 36degrees** [comment]
Appears to be fixed as of (at least) iOS 12.2 (16E227).
